pkg/chartutil/create.go

```diff
@@ -149,6 +151,13 @@ resources: {}
   #   cpu: 100m
   #   memory: 128Mi
 
+autoscaling:
+  enabled: false
```

Comment:
This line is also important. I think optional - but good to have templated - components like this should be `false` by default.